syntax = "proto3";

service Communication {
  rpc SendNumbers(stream NumbersRequest) returns (NumbersResponse) {}
}

message NumbersRequest {
  float number1 = 1;
  float number2 = 2;
  float number3 = 3;
  float number4 = 4;
}

message NumbersResponse {
  string message = 1;
}